dlopen the sonamed libs directly, like it has already been done for libGL.so.1,
as we don't want to Depend on the -dev packages for the .so files:
- libosmesa.so.6: was already available in Hardy
- libGLESv2.so.1 and libEGL.so.1: available only since Maverick

---
 ui/gfx/gl/gl_implementation_linux.cc |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/ui/gl/gl_implementation_linux.cc
+++ b/ui/gl/gl_implementation_linux.cc
@@ -75,7 +75,7 @@
       }
 
       base::NativeLibrary library = LoadLibrary(
-          module_path.Append("libosmesa.so"));
+          module_path.Append("libosmesa.so.6"));
       if (!library)
         return false;
 
